src/cephadm/cephadm

```diff
@@ -9562,6 +9562,7 @@ def cephadm_init_logging(ctx: CephadmContext, args: List[str]) -> None:
     compress
     missingok
     notifempty
+    su root root
```

Comment:
I'm wondering if there's any possibility of running this a non privileged user?

Reply:
cephadm itself will always be running as root because we require it https://github.com/ceph/ceph/blob/60febf5b0d42289e651e3ab9703a1faa70d1873a/src/cephadm/cephadm#L9575-L9579 so the log itself will be root owned, and we already use this for the other logs we support logrotation for https://github.com/ceph/ceph/blob/60febf5b0d42289e651e3ab9703a1faa70d1873a/src/cephadm/cephadm#L3734-L3744 which I've heard no complaints about. Also, for this to work, the directory or log file itself don't have to be owned by group root, we just need it to use a user/group with high enough permissions for it to rotate the log and you can't get higher than root/root